test/integration/acceptance/set_account_detail_test.cpp

```diff
@@ -147,3 +167,69 @@ TEST_F(SetAccountDetail, WithGrantablePerm) {
         ASSERT_EQ(block->transactions().size(), 1);
       });
 }
+
+/**
+ * C276
+ * @given a user with required permission
+ * @when execute tx with SetAccountDetail command with max key
+ * @then there is no tx in proposal
+ */
+TEST_F(SetAccountDetail, HugeKey) {
+  IntegrationTestFramework(1)
+      .setInitialState(kAdminKeypair)
+      .sendTx(makeUserWithPerms())
+      .skipProposal()
+      .skipBlock()
+      .sendTxAwait(
+          complete(baseTx(kUserId, std::string(64, 'a'), kValue)),
```

Comment:
Please move `std::string(64, 'a')` as a const var to the beginning of the test case. 
The reason is the same as in the first review comment.